osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -116,9 +116,10 @@ protected override void LoadComplete()
             new KeyBinding(new[] { InputKey.F3 }, GlobalAction.DecreaseScrollSpeed),
             new KeyBinding(new[] { InputKey.F4 }, GlobalAction.IncreaseScrollSpeed),
             new KeyBinding(new[] { InputKey.Shift, InputKey.Tab }, GlobalAction.ToggleInGameInterface),
+            new KeyBinding(InputKey.Tab, GlobalAction.ToggleInGameLeaderboard),
             new KeyBinding(InputKey.MouseMiddle, GlobalAction.PauseGameplay),
             new KeyBinding(InputKey.Control, GlobalAction.HoldForHUD),
-            new KeyBinding(InputKey.Tab, GlobalAction.ToggleChatFocus),
+            new KeyBinding(InputKey.Enter, GlobalAction.ToggleChatFocus),
```

Comment:
Dunno about this being enter. I can't outright break it but I wouldn't say it's intuitive for enter to toggle chat.

Reply:
I chose enter since it was common for other multiplayer games like overwatch, val, and league. I'm super open to changing it to something else.

Reply:
I think enter is probably okay.